Formula/wasm4.rb

```diff
@@ -0,0 +1,66 @@
+require "language/node"
+
+class Wasm4 < Formula
+  desc "Low-level fantasy game console for building small games with WebAssembly"
+  homepage "https://wasm4.org/"
+  url "https://github.com/aduros/wasm4.git",
+      tag:      "v2.5.3",
+      revision: "9d782a5c0465e54ec8a4b95d21e4e8a10fdd9ae2"
+  license "ISC"
+
+  depends_on "cmake" => :build
+  depends_on "node"
+
+  on_linux do
+    depends_on "alsa-lib"
+    depends_on "libxcursor"
+    depends_on "libxext"
+    depends_on "libxi"
+    depends_on "libxinerama"
+    depends_on "libxkbcommon"
+    depends_on "libxrandr"
+    depends_on "pulseaudio"
+  end
+
+  def install
+    cd "runtimes/native" do
+      system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+      system "cmake", "--build", "build"
+      if OS.mac?
+        (buildpath/"cli/assets/natives").install "build/wasm4" => "wasm4-mac"
+      else
+        (buildpath/"cli/assets/natives").install "build/wasm4" => "wasm4-linux"
+      end
```

Comment:
no cmake `--install`?

Reply:
Why would it need to be renamed?

Reply:
agree, we dont have to rename the artifacts?

Reply:
You can make a symlink. But additionally it's not up to Homebrew to fix upstream issues.